### ISSUE TITLE
Inversion of wind speed and direction

### ISSUE THREAD

**Issue by Grandasse** [LesterGallagher/atc-manager-2#40]
Hello, 

I noticed in the [getCommandText function](https://github.com/sempostma/atc-manager-2/blob/64fe4758d31ec19f88ae46cdc3518e97f8178c8a/lib/communications.js#L55C57-L55C67), we can see that wind speed and direction are inverted. 

The sentence should be `" the wind is ${winddirtxt} at ${windspdtxt} knots"` instead.

**Comment by Creative-Difficulty** [comment]
you could try to open a PR!

**Comment by Grandasse** [comment]
Indeed, I did it here https://github.com/sempostma/atc-manager-2/pull/42

**Comment by sempostma** [comment]
Sorry for the late reply, this is now merged
